src/App.vue

```diff
@@ -14,6 +14,9 @@ export default defineComponent({
 </script>
 
 <style lang="scss">
+* {
```

Comment:
although this might work as a fix preventing the x overflow, it sets the max-width property on literally all elements, instead of fixing the (probably) single element that causes the x overflow, consider fixing the element that causes it instead

Reply:
done, not needed